Dockerfile

```diff
@@ -40,6 +40,5 @@ RUN yarn install --check-files
 # if we override settings at runtime to be the local-dev settings, we'll build
 # and use local-dev assets.
 RUN yarn prod
-RUN python manage.py collectstatic --noinput
```

Comment:
Because it appears that the `--env` or `--env-file` flags implicitly passed in by `docker-compose` from the configuration yaml aren't available except at the `CMD` step. So the Django settings are wrong at this stage, and fail. But as I write this, I realize that, since `collectstatic` doesn't require a GitHub API key, I could just set a default on it in the settings and let that get overridden at the point of actual server start. Making that change now.